integrationtests/pom.xml

```diff
@@ -37,18 +37,6 @@
             <version>0.2.0-SNAPSHOT</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
```

Comment:
so - just making sure I understand the direction.  integrationtests module - private concern of the kroxylicious project for testing broad features of the component. kroxylicious-test-tools - something that  kroxy's plugin modules /3rd parties can consume to get the ability to integration test their kroxy filters).  Am I on the right lines?

Reply:
Correct, though I've moved the multitenant and json validation ITs back into `integrationtests` for now. But that's the essence of it, `kroxylicious-test-tools` should be a library users can depend on to integration test filters. Maybe renaming it to `kroxylicious-test` would be a more standard naming.

Reply:
I've also purposefully not added any Junit integration as it looked like a rabbithole to figure out how to make two extensions play nicely together. Maybe we could find a way later.